data-structures/big-o/description.ts

```diff
@@ -0,0 +1,170 @@
+// TODO: change images
```

Comment:
It would be cool to be able to use markdown files (*.md) some day, I was even playing around, because they give an example in next documentation but didn't have time to finish that